internal/provider/resource_hvn.go

```diff
@@ -323,11 +324,35 @@ func setHvnResourceData(d *schema.ResourceData, hvn *networkmodels.HashicorpClou
 // resourceHvnImport implements the logic necessary to import an un-tracked
 // (by Terraform) HVN resource into Terraform state.
 func resourceHvnImport(ctx context.Context, d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
+	// with multi-projects, import arguments must become dynamic:
+	// use explicit project ID with terraform import:
+	//   terraform import hcp_hvn.test f709ec73-55d4-46d8-897d-816ebba28778:test-hvn
+	// use default project ID from provider:
+	//   terraform import hcp_hvn.test test-hvn
+
 	client := meta.(*clients.Client)
+	projectID := ""
+	hvnID := ""
+	var err error
+
+	if strings.Contains(d.Id(), ":") {
```

Comment:
This works here, but this conditional will have to change for resources that already have multiple parts (i.e. HVN route: https://github.com/hashicorp/terraform-provider-hcp/blob/main/internal/provider/resource_hvn_route.go#L295)